scripts/http-ok.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env node
+
+// This script sets up a very simple http server responding to /health with a 200 OK.
+
+const http = require('http');
+
+http.createServer(function (req, res) {
+    if(req.url === "/health") {
+        res.writeHead(200);
+        res.end(JSON.stringify({
+            success: true,
+            server: "healthcheck-ok",
+        }));
+        return;
+    }
+    res.writeHead(500);
+    res.end(JSON.stringify({
+        success: false,
+        server: "healthcheck-ok",
+        message: "This service is only meant to be queried at /health"
+    }));
+}).listen(process.env.PORT);
```

Comment:
`process.env.PORT` is undefined in the watch container - just needs to be added to the environment in the docker-compose but this does cause it to not build when things are set to production (I think - that worked for me, at least to get it to start up).